### PR TITLE
Maintenance

### PR DESCRIPTION
**Pull request by TacoTheDank**
yes

### CODE REVIEW

File: app/src/main/java/com/tasomaniac/betachecker/ShareToCheckBeta.java [hunk 1]
```diff
@@ -30,13 +30,13 @@ protected void onCreate(Bundle savedInstanceState) {
             return;
         }
 
-        Intent intent = new CustomTabsIntent.Builder()
```
Comment (tasomaniac) [inline app/src/main/java/com/tasomaniac/betachecker/ShareToCheckBeta.java]: Is there a reason for this rename? If not necessary can you revert it?

**Review by tasomaniac — COMMENTED** [review]
Nice! Thank you for taking time for this. I would be happy to try to jettify the app and use androidx dependencies. If you are interested, that's a nice maintenance task.

File: app/src/main/java/com/tasomaniac/betachecker/ShareToCheckBeta.java (hunk 1 above)
Comment (TacoTheDank) [reply app/src/main/java/com/tasomaniac/betachecker/ShareToCheckBeta.java]: kk

**Comment by TacoTheDank** [comment]
i would do that but im not familiar with working with androidx ¯\\\_(ツ)_/¯

**Review by tasomaniac — APPROVED** [review]

**Pull request merged**